docs/content/getting-started.md

```diff
@@ -211,6 +236,7 @@ type Todo struct {
 	ID     string `json:"id"`
 	Text   string `json:"text"`
 	Done   bool   `json:"done"`
+	UserID string `json:"userId"`
```

Comment:
hello @a70537952 , I also had this problem but it has been fixed with your guidance here.
may I know why you adding new UserID in this line?

Reply:
Code Snippet from Getting Started doc
```go
func (r *mutationResolver) CreateTodo(ctx context.Context, input model.NewTodo) (*model.Todo, error) {
	todo := &model.Todo{
		Text:   input.Text,
		ID:     fmt.Sprintf("T%d", rand.Int()),
		User:   &model.User{ID: input.UserID, Name: "user " + input.UserID},
	}
	r.todos = append(r.todos, todo)
	return todo, nil
}

func (r *todoResolver) User(ctx context.Context, obj *model.Todo) (*model.User, error) {
	return &model.User{ID: obj.UserID, Name: "user " + obj.UserID}, nil  //This line
}
```
Fix missing `UserID` field from model.Todo

Reply:
UserID is on input -- I don't think this should be added -- can you look again @a70537952 ?

Reply:
@tmc if I'm not wrong, this User resolver is for Query which related to `model.Todo userID` , nothing to do with UserID on input `model.NewTodo userID`